src/js/global.d.ts

```diff
@@ -1,4 +1,4 @@
 declare module "@stackoverflow/stacks-icons" {
-    export * as Icons from "@stackoverflow/stacks-icons/icons";
-    export * as Spots from "@stackoverflow/stacks-icons/spots";
+    export namespace Icons {}
+    export namespace Spots {}
```

Comment:
Got a bit stuck here with an error of the package not being found before build, not sure if this breaks anything or not

Reply:
Hmm that's odd. I can't reproduce that locally and the original exports make the icon and spot types available when developing around the codebase.

Is this reported from a regular `npm start`/`npm run build`?

Reply:
Just want to recommend caution with changing those declaration files. They could cause unexpected type errors for consumers down the line. If changes need to be made here I would recommend to manually test them in Core before merging to see how the TS compiler reacts there. 🙂

Reply:
You can replicate by deleting your `dist` folder to simulate a fresh build.

I've figured it out by using `@ts-expect-error`, which is what we use in the other files in this folder. Should be good now!